shared/src/main/java/uk/gov/di/authentication/shared/entity/ClientSession.java

```diff
@@ -16,6 +19,7 @@ public class ClientSession {
     @Expose private LocalDateTime creationDate;
 
     @Expose private VectorOfTrust effectiveVectorOfTrust;
+    @Expose private List<VectorOfTrust> vtrList = new ArrayList<>();
```

Comment:
The inconsistent space here makes me itch slightly, but I don't mind that much

Reply:
Oh yeah that's really not nice, good spot